extensions/positron-zed/src/commands.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+export async function registerCommands(context: vscode.ExtensionContext) {
+	context.subscriptions.push(
+
+		vscode.commands.registerCommand('zed.quartoVisualMode', () => {
+			const editor = vscode.window.activeTextEditor;
+			if (!editor) {
+				return;
+			}
+			vscode.commands.executeCommand('positron.reopenWith', editor.document.uri, 'quarto.visualEditor');
```

Comment:
This is how the Quarto extension can use the new command. To go the other way, one would do:

```typescript
vscode.commands.executeCommand('positron.reopenWith', editor.document.uri, 'default');
```

There isn't a good easy way to do this in the Zed extension, though, because Zed doesn't have a quick way to get to the active visual editor. The [Quarto extension already keep tracks of all that](https://github.com/quarto-dev/quarto/blob/491057ba2005584b48ed511cf82f49d3b630362f/apps/vscode/src/providers/editor/toggle.ts#L91), naturally.